src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let acc = 0;
+
+  const add = (num) => {
+    if (num) {
```

Comment:
if you call `adder(0)` your program will work incorrectly.
Let's change the condition check.